.gitignore

```
@@ -0,0 +1,2 @@
+.env
+open_prs.csv
```

```suggestion
env/
ENV/
```
And save the output to either of those directories locally?

---

Good call. I've moved everything to a new dir, and a separate .gitignored dir for the outputs in 56cacb8a